immutable_set/immutable_set.mbt

```diff
@@ -558,38 +551,31 @@ pub fn filter[T : Compare](
   }
 }
 
-/// Find value in self and return value if present, otherwise abort.
+/// Find value in self and return value if present with Option, otherwise None.
 /// 
 /// # Example
 /// 
 /// ```
-/// println(ImmutableSet::from_list(List::[1, 2, 3, 4, 5, 6]).find(6))
-/// // output: 6
+/// println(ImmutableSet::[1, 2, 3, 4, 5, 6].find(6))
+/// // output: Some(6)
```

Comment:
this API `find` is not very useful for `set`, 
either remove it or retain the `exact` element in the `Set`